README.md

```diff
@@ -21,7 +21,7 @@ See the [CHANGELOG](https://github.com/webcompat/webcompat.com/blob/master/CHANG
 
 ## Contributing
 
-Contributions in the form of code, feature requests, bugs or other kinds of issues are welcome. Please check out [CONTRIBUTING.md]( https://github.com/webcompat/webcompat.com/blob/master/CONTRIBUTING.md) for more instructions. Everyone interacting in our codebases, issue trackers, and any other form of communication, including chat rooms and mailing lists, is expected to follow our [code of conduct](https://github.com/webcompat/webcompat.com/blob/master/CODE_OF_CONDUCT.md) so we can all enjoy the effort we put into this project.
+Contributions in the form of code, feature requests, bugs or other kinds of issues are welcome. Please check out [CONTRIBUTING.md]( https://github.com/webcompat/webcompat.com/blob/master/CONTRIBUTING.md) for more instructions, including how to set up a [local build environment](https://github.com/webcompat/webcompat.com/blob/master/docs/dev-env-setup.md). Everyone interacting in our codebases, issue trackers, and any other form of communication, including chat rooms and mailing lists, is expected to follow our [code of conduct](https://github.com/webcompat/webcompat.com/blob/master/CODE_OF_CONDUCT.md) so we can all enjoy the effort we put into this project.
```

Comment:
Were writing new content right now, which you can see in the refactor branch.
We openened our definition of contribution a bit more as you can see here:
https://github.com/webcompat/webcompat.com/blob/refactor/webcompat/templates/contact.html#L50

It would be great, if you could either include a more open wording for "in the form of code". Maybe change it to something like "All kinds of contributions are welcome like code, design, writing, feature request..."

Thanks!